signs-work-order-map/app/calcs/page.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import { useForm } from "react-hook-form";
+import CalcWidthLengthComponent from "@/components/CalcWidthLengthComponent";
+import CalcRPMSComponent from "@/components/CalcRPMSComponent";
+import { CalculationInputsType } from "@/types/calcs";
+import { thermoplasticCalculations, paintCalculations } from "@/config/calcs";
+import "./calc.css";
+
+const defaultCalcValues = {
+  thermo60Width: 0,
+  thermo60LinearFeet: 0,
+  thermo60Output: 0,
+  thermo90Width: 0,
+  thermo90LinearFeet: 0,
+  thermo90Output: 0,
+  beadsExtrudedWidth: 0,
+  beadsExtrudedLinearFeet: 0,
+  beadsExtrudedOutput: 0,
+  primerWidth: 0,
+  primerLinearFeet: 0,
+  primerOutput: 0,
+  paintGallonsWidth: 0,
+  paintGallonsLinearFeet: 0,
+  paintGallonsThickness: 0,
+  paintGallonsOutput: 0,
+  beadsPaintWidth: 0,
+  beadsPaintLinearFeet: 0,
+  beadsPaintOutput: 0,
+  adhesiveInputRPMS: 0,
+  adhesiveOutput: 0,
+};
+
+export default function Calcs() {
+  const { control, watch, setValue } = useForm<CalculationInputsType>({
```

Comment:
i've been thinking about how you might simplify the configs, and one option could be to move the `<form>`/`useForm` hook into the `CalcWidthLengthComponent`. Since these different sections don't rely on each other in any way. I think that'd let you use the same field names for each instance of the form and not have to track so many variables and what have you. Just a suggestion based on my quick scan of the code, so if that sounds appealing, great, otherwise please ignore 👍